### PR TITLE
AAUser on Captain staffing, when i look at 'NOTHERE' project a dev is on i see it in bold as 'Not Started'.

### PR DESCRIPTION
**Pull request by SimonLTheodo**
https://trello.com/c/jqXPJDiC/53-3-aauser-on-captain-staffing-when-i-look-at-nothere-project-a-dev-is-on-i-see-it-in-bold-as-not-started

### CODE REVIEW

File: src/components/staffing/ProjectCell.js [hunk 1]
```diff
@@ -29,11 +29,22 @@ export default class ProjectCell extends React.Component {
...
+    const currentProjectsText = renamedCurrentProjects.join(', ')
     const separator = currentProjects.length > 0 && upcomingProjects.length > 0 ? ', ' : ''
     const upcomingProjectsText = upcomingProjects.join(', ')
     const isIndividualProject = data[rowIndex].projects === undefined
-    const cellText = isIndividualProject ? data[rowIndex][field] : (
+    const individualProjectText = isIndividualProject ? data[rowIndex][field] : ''
+    const individualProjectRenamedText = MAP_PROJECT_NAMES[individualProjectText]
+    ? MAP_PROJECT_NAMES[individualProjectText]
+    : individualProjectText
+    const cellText = isIndividualProject ? individualProjectRenamedText : (
```
Comment (Francisco-Costa98) [inline src/components/staffing/ProjectCell.js:47]: i know you didnt write this code but would be better if this was a component

**Review by Francisco-Costa98 — APPROVED** [review]

File: src/components/staffing/ProjectCell.js (hunk 1 above)
Comment (SimonLTheodo) [reply src/components/staffing/ProjectCell.js]: Fixed!

File: src/components/staffing/ProjectCellText.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { MAP_PROJECT_NAMES } from '../../constants'
+
+export default class ProjectCellText extends React.Component {
```
Comment (Francisco-Costa98) [inline src/components/staffing/ProjectCellText.js:4]: im assuming youve done a class component for consistency ?

Comment (SimonLTheodo) [reply src/components/staffing/ProjectCellText.js:4]: Yep

**Pull request merged**